### DISCUSSION TITLE
Pages with dynamic routes often lead to 404 on hot reload

### DISCUSSION THREAD

**Discussion by hragbalian** [reflex-dev/reflex#3465] [Q&A]
Hello,

I'm running into a strange issue where pages with regular dynamic routes often lead to 404 on hot reload.  By often, I would say it's about 40-50% of my reloads result in this outcome... and when it occurs, it almost always resolves on a subsequent reload, with zero changes to my code.  The issue only occurs on pages with dynamic routes, and I've never observed it on static routes. 

I'm wondering if this is a known issue and if there's a workaround.  

Thanks!

**Comment by masenf** [comment]
Also reported in #3416 and i've anecdotally hit it myself while developing the "form-designer" sample app which makes extensive use of dynamic routes.

Unfortunately I don't know of a workaround for it, other than just manually restarting reflex when it occurs.

Needs investigation still.
